inst/report/scripts/generate_report.R

```diff
@@ -0,0 +1,705 @@
+                                                    da_iprVisType,
+                                                    DASelect)
+
+    query_heatmap_value <- generate_query_heatmap(query_data@df,
+                                                  heatmap_select = "All",
+                                                  heatmap_color = "blue")
+
+    DA_Prot_value <- get_DA_Prot(app_data, DASelect)
+
+    DALinPlot_value <- generate_DA_heatmap_plot(DA_col = "DomArch.Pfam",
+                                                DACutoff,
+                                                DA_Prot_value,
+                                                DA_lin_color = "viridis",
+                                                app_data@ipr_path)
+
+    DALin_TotalCounts_value <- DA_TotalCounts(DA_Prot_value,
+                                              DACutoff,
+                                              DA_col = "DomArch.Pfam",
+                                              app_data)
+
+    DALinTable_value <- generate_DA_lin_table(DA_col = "DomArch.Pfam",
+                                              app_data@ipr_path,
+                                              DALin_TotalCounts_value)
+
+    DANetwork_value <- generate_domain_network(DA_col = "DomArch.Pfam",
+                                               DACutoff,
+                                               DA_Prot_value,
+                                               networkLayout = "nice",
+                                               app_data@ipr_path)
+
+    rep_accnums_value <- rep_accnums(phylo, msa_reduce_by, msa_rep_num, PhyloSelect, app_data)
+
+    phylogeny_prot_value <- filter_phylogeny_proteins(app_data, phylo_select)
+
+    acc_to_name_value <- acc_to_name(app_data)
+
+    ####### Report Generation ########
+
+    tryCatch({
+        tempReport <- file.path(tempdir(), "report.Rmd")
+        file.copy(report_template_path, tempReport, overwrite = TRUE)
+
+        # List of graphics to include in report
+        params <- list(
+            rs_interproscan_visualization = rs_IprGenes_value,
+            proximity_network = rs_network_layout_value,
+            sunburst = data@df,
+            data = rs_data_table_value,
+            queryDataTable = queryDataTable_value,
+            fastaDataText = fastaDataText_value,
+            heatmap = query_heatmap_value,
+            query_data = query_data,
+            query_domarch_cols = query_domarch_cols_value,
+            query_iprDatabases = query_iprDatabases,
+            query_iprVisType = query_iprVisType,
+            mainTable = mainTable_value,
+            DALinTable = DALinTable_value,
+            DALinPlot = DALinPlot_value,
+            DANetwork = DANetwork_value,
+            DA_Prot = DA_Prot_value,
+            domarch_cols = domarch_cols_value,
+            DA_Col = DA_Col,
+            DACutoff = DACutoff,
+            da_interproscan_visualization = da_IprGenes_value,
```

Comment:
Please check for name consistency, `interpro` vs. `IPR` in caps vs. `ipr` in small.
Also, there's `vis` in some places and `viz` in others.
cc: @the-mayer @epbrenner probably have looked through enough other functions to know which would work most consistently. 

```suggestion
            da_iprscan_viz = da_iprGenes_value,
```